### ISSUE TITLE
Usuń "subtract fee from amount" oraz zmień pozycję i opis "Send all funds"

### ISSUE THREAD

**Issue by Fl4reTobiasH** [Fl4reTobiasH/HDL#15]
Subtract fee from amount - do wyrzucenia z gui, przy opcji send all funds ta opcja będzie użyta automatycznie, a zwykły przelew nie używa tej opcji. 

Send all funds, tak jak na poniższym obrazku:
![image](https://user-images.githubusercontent.com/45454639/53878426-21101c80-400c-11e9-93f5-4b37c3eca6b7.png)

**Comment by zurekpiotr** [comment]
Zostawiłbym jakieś ostrzeżenie że fee zostanie pobrane z wysyłanej kwoty.
trochę słabo by było jakby ktoś miał zapłacić 15USD   na koncie miał 16 a automatyczne fee wyszło 2.

**Comment by tendersoft-mjb** [comment]
Czy przy wysyłaniu transakcji pojawia się ekran z potwierdzeniem, który zawiera wszystkie informacje?

**Comment by zurekpiotr** [comment]
Jest ekran z podsumowaniem transakcji i hasłem do jej potwierdzenia.

**Comment by tendersoft-mjb** [comment]
to jeżeli tam jest widoczna i kwota, i fee, to jakiegoś dodatkowego ostrzeżenia bym nie generował - w koncu po to jest podsumowanie, zeby wlasnie sprawdzic czy się wszystko zgadza.

Natomiast warto dodać np. **pogrubienie** dla 4-ech pierwszych i 4-ech ostatnich znaków adresu, na który idzie płatność - np. w aplikacji mBanku jest coś takiego dla numeru konta i jest to bardzo wygodne pod względem zachowania bezpieczeństwa transakcji.

przykład: **3J98**t1WpEZ73CNmQviecrnyiWrnqRh**WNLy**

**Comment by Fl4reTobiasH** [comment]
@tendersoft-mjb To już mamy w najnowszej wersji portfela (pogrubione i pokolorowane znaki w adresie) 

![image](https://user-images.githubusercontent.com/45454639/54112523-a74ea900-43e6-11e9-883a-fc074507f51d.png)

Odnośnie ostrzeżenia, myślę że jest niepotrzebne. Opcja Subtract fee byłaby aktywna wyłącznie w momencie kliknięcia "wyślij wszystko". Ta opcja służy raczej do przeniesienia salda a nie do płatności za coś. @zurekpiotr  płaciłeś kiedyś za coś opcją "Send All"? :)

**Comment by tendersoft-mjb** [comment]
OK, przy czym to jest w historii. Adres w podsumowaniu przed potwierdzeniem transakcji, także ma kolorki? :) @Fl4reTobiasH masz rację, co do sposobu użycia tego Subtract fee - automat przy całej kwocie, a tak to samo się dolicza - tak to wszędzie działa, więc użytkownicy oczekują takiego właśnie sposobu konstuowania transakcji. Dodatkowa opcja jest zbędna, tym bardziej że zawsze to jakieś potencjalne pole do błędów.

**Comment by Fl4reTobiasH** [comment]
@tendersoft-mjb Racja, w podsumowaniu nie ma kolorowania. Samo podsumowanie wymaga odrębnego opracowania żeby było bardziej czytelne. Przygotuję je.
